proxy/src/proxy_config.cfg

```diff
@@ -80,6 +80,19 @@ frontend haproxy_v4_xmpp
 
   default_backend wa
 
+frontend haproxy_v4_whatsapp_net
+  maxconn 27495
+  #PUBLIC_IP
+
+  bind ipv4@*:587
+  bind ipv4@*:7777
+
+  default_backend wa_whatsapp_net
+
+backend wa_whatsapp_net
+  default-server check inter 60000 observe layer4
+  server whatsapp_net_443 whatsapp.net:443
```

Comment:
I feel like we should update the [healthchecker](https://github.com/WhatsApp/proxy/blob/main/proxy/src/healthcheck.sh) as well?

Reply:
Done! Let me know what you think!

Reply:
#217